core/storage/statistics/gae_models.py

```diff
@@ -52,6 +52,15 @@ class StateCounterModel(base_models.BaseModel):
 
     @classmethod
     def get_or_create(cls, exploration_id, state_name):
+        """Gets or creates an entity by exploration_id and state_name.
+
+        Args:
+            exploration_id: str. ID of the exploration currently being played.
+            state_name: str. Name of the current state.
+
+        Returns:
+            StateCounterModel. A model instance of the StateCounterModel.
```

Comment:
Nit: An instance of the StateCounterModel.

Reply:
edited